python/py-acoustid/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
```

Comment:
this doesn't need/use the github PG anymore; other than that: looks good!

Reply:
@reneeotten very good catch! Thanks!